client/app/lib/remote-extensions/machine.coffee

```diff
@@ -189,6 +200,38 @@ module.exports = class JMachine extends remote.api.JMachine
 
       callback err, machine
 
+  getStatus: -> @getAt 'status.state'
+
+
+  getSharedUsers: -> @sharedUsers or []
+
+  reviveUsers: (options, callback = kd.noop) ->
+
+    { storage } = kd.singletons.computeController
+
+    super options, (err, users = []) =>
+      return callback err  if err
+
+      @sharedUsers = users
```

Comment:
better to update `@users` instead, with something like: `{username: foo, instance: JAccountOf 'foo'}`

`sharedUsers` is not a property of `JMachine`